### PR TITLE
Expose Tor Version from gradle.properties

### PR DESCRIPTION
**Pull request by rodvar**
- Implementation of https://github.com/bisq-network/bisq2/issues/828
 - move tor version definition to root gradle.properties
 - do necessary changes to read from root in current tor version usages
 - expose tor version in generated BuildVersion Java class
 - print tor version along with bisq version on startup logs
 - add label to network info view with tor version

### CODE REVIEW

File: application/src/main/java/bisq/application/ApplicationService.java [hunk 1]
```diff
@@ -156,6 +156,8 @@ public ApplicationService(String configFileName, String[] args) {
         log.info(AsciiLogo.getAsciiLogo());
         log.info("Data directory: {}", config.getBaseDir());
         log.info("Version: v{} / Commit hash: {}", ApplicationVersion.getVersion().getVersionAsString(), ApplicationVersion.getBuildCommitShortHash());
+        log.info("Tor Version: v{}", ApplicationVersion.getTorVersionString());
```
Comment (HenrikJannsen) [inline application/src/main/java/bisq/application/ApplicationService.java:159]: Maybe just add it to the previous log?

File: apps/desktop/desktop/src/main/java/bisq/desktop/main/content/settings/network/NetworkInfoView.java [hunk 2]
```diff
@@ -54,9 +54,11 @@ public NetworkInfoView(NetworkInfoModel model, NetworkInfoController controller,
         Label myVersionAndCommitHash = new Label(Res.get("settings.network.myVersionAndCommitHash",
                 ApplicationVersion.getVersion().getVersionAsString(),
                 ApplicationVersion.getBuildCommitShortHash()));
+        Label torVersion = new Label(Res.get("settings.network.torVersion",
```
Comment (HenrikJannsen) [inline apps/desktop/desktop/src/main/java/bisq/desktop/main/content/settings/network/NetworkInfoView.java]: Better add it to the `settings.network.myVersionAndCommitHash` string. As it is now, the vertical distance is different between the first 2 lines and tor version as first 2 use a linebreak in the string.

**Review by HenrikJannsen — CHANGES_REQUESTED** [review]
Some small issues. The layout is not ok as it is in version display. As mentioned best to append the tor version with a linebreak to existing label

File: application/src/main/java/bisq/application/ApplicationService.java (hunk 1 above)
Comment (rodvar) [reply application/src/main/java/bisq/application/ApplicationService.java:159]: I think is clearer this way

File: apps/desktop/desktop/src/main/java/bisq/desktop/main/content/settings/network/NetworkInfoView.java (hunk 2 above)
Comment (rodvar) [reply apps/desktop/desktop/src/main/java/bisq/desktop/main/content/settings/network/NetworkInfoView.java]: the space was left on purpose, but we discussed on matrix and agree to try to put them together which I'm gonna do now

**Review by HenrikJannsen — APPROVED** [review]
utACK

**Pull request merged**